### PR TITLE
bpo-28955: Clarified comparisons between NaN and number in reference documentation

### PR DESCRIPTION
**Pull request by TonyFlury**
… documentation

https://bugs.python.org/issue28955

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

**Comment by TonyFlury** [comment]
CLA signed - 11:10am GMT - 4th March 2018

File: Misc/NEWS.d/next/Documentation/2018-03-05-08-33-00.bpo-28955.-OSvYJ.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Issue 28955 - Clarified numeric comparisons with NaN. Patch by Tony Flury
```
Comment (berkerpeksag) [inline Misc/NEWS.d/next/Documentation/2018-03-05-08-33-00.bpo-28955.-OSvYJ.rst]: We don't require news entries for trivial documentation changes. You can delete this file.

File: Doc/reference/expressions.rst [hunk 2]
```diff
@@ -1336,10 +1336,10 @@ built-in types.
 
   The not-a-number values :const:`float('NaN')` and :const:`Decimal('NaN')`
   are special.  They are identical to themselves (``x is x`` is true) but
-  are not equal to themselves (``x == x`` is false).  Additionally,
-  comparing any number to a not-a-number value
+  are not equal to themselves (``x == x`` is false), or any other number.
+  Additionally, all ordered comparison of any number to a not-a-number value
   will return ``False``.  For example, both ``3 < float('NaN')`` and
-  ``float('NaN') < 3`` will return ``False``.
+  ``float('NaN') < 3`` will return ``False``. This is compliant with IEE 754.
```
Comment (berkerpeksag) [inline Doc/reference/expressions.rst]: Shouldn't this be IEEE 754?

Comment (berkerpeksag) [inline Doc/reference/expressions.rst]: Style nit: Please start a new sentence with two spaces. See line 1338 for example:

```rst
...
are special.  They are identical to themselves (``x is x`` is true) but
...
```

**Review by berkerpeksag — CHANGES_REQUESTED** [review]

File: Doc/reference/expressions.rst (hunk 2 above)
Comment (TonyFlury) [reply Doc/reference/expressions.rst]: Agreed.

**Comment by TonyFlury** [comment]
I have made the requested changes; please review again.

**Review by berkerpeksag — APPROVED** [review]

**Pull request merged**